lighthouse-core/test/runner-test.js

```diff
@@ -633,27 +626,6 @@ describe('Runner', () => {
     });
   });
 
-
-  it('rejects when not given a URL', async () => {
```

Comment:
the benefit of these here is that it ensured the behavior of `Runner.run()` regardless of where the URL checking occurs (living more on the "integration" side of the integration/unit divide). Are there still tests checking that legacy (and now `navigation-runner`) are checking these states?

Reply:
There are some in https://github.com/GoogleChrome/lighthouse/blob/master/lighthouse-core/test/index-test.js.